tools/license-check.sh

```diff
@@ -12,6 +12,11 @@ do
     echo "ERROR: Package $name does not have a license"
     errors=true
   fi
+  license=$(grep -Po 'license: \K.*' $file| tr -d '"')
+  if echo $license | grep -vqEx 'Apache-1.0|Apache-2.0|BSD-2-Clause|BSD-3-Clause|curl|GPL-1.0-or-later|GPL-2.0-only|GPL-3.0-only|ISC|MIT|MPL-2.0|PSF-2.0|LGPL-2.0-or-later|LGPL-2.1-only|LGPL-2.0-only WITH OCaml-LGPL-linking-exception|LGPL-2.0-or-later WITH OCaml-LGPL-linking-exception|LGPL-2.1-only WITH OCaml-LGPL-linking-exception|LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception|LGPL-3.0-only WITH OCaml-LGPL-linking-exception|LGPL with OpenSSL linking exception|GPL-2.0-only AND LGPL-3.0-only|MIT AND LGPL-2.1-only WITH OCaml-LGPL-linking-exception'; then 
```

Comment:
I would prefer the licenses in a list but I don't have a good idea to do this in a shell script. One way would be to store them in a file and use `grep -f`

Reply:
I've put the licenses into file, hopefully this is more palatable now